cfme/tests/configure/test_access_control.py

```diff
@@ -2816,7 +2815,11 @@ def test_self_service_users_ssui():
             1. User created successfully
             2. SSUI access granted
     """
-    pass
+    user, _ = user_self_service_role
+
+    with user:
+        with appliance.context.use(ViaSSUI):
+            appliance.server.login(user)
```

Comment:
It kinda looks like a smoke test. If you think so too, better to add a smoke test marker.